nnoir-onnx/nnoir_onnx/operators/utils.py

```diff
@@ -23,11 +25,15 @@ def __init__(self, message):
 
 class Op:
 
-    def __init__(self, node, opset_version):
+    def __init__(self, node, opset_version, onnx):
         self.node = node
         self.opset_version = opset_version
+        self.onnx = onnx
 
-    def to_function(self, env, constants):
+    def to_function(self, env, constants) -> Tuple[List["Op"], List[Value]]:
+        """
+        return the list of operators and values representing intermediate nodes
+        """
```

Comment:
I modify all the signature of `to_function` from `List[Op]` to `Tuple[List[Op], List[Value]]`.
Because of the complexity of the type, we need type signature here.

Reply:
I revert it in 3da22eb94585a3dbb323ef6a5d0c11769c57fd5d